mautic/mautic
# Use new authenticator system.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | red 
| New feature/enhancement? (use the a.x branch)      | new
| Deprecations?                          | yes
| BC breaks? (use the c.x branch)        | yes
| Automated tests included?              | not for new functionality 



## Description

Remove deprecated authentication of Symfony 5. Use other forks to address the issues with current and next Symfony versions.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test general login
3. Test SAML solgin - instructions for setting up a local SAML testing facility can be found here: https://github.com/mautic/mautic/pull/13742#issue-2291225011
4. Test API login (mainly done by tests)
5. Test SSO login
6. Enable saml, and check if regular login page is available. Remove a todo in \Mautic\UserBundle\Security\EntryPoint\MainEntryPoint::start
